portability-types-common/src/main/java/org/datatransferproject/types/common/models/photos/PhotoModel.java

```diff
@@ -94,14 +95,28 @@ public String getDescription() {
     return description;
   }
 
+  // TODO(zacsh) remove this in favor of getFolderId
```

Comment:
Is getFolderId the right method to replace this? Based on the name I would think mediaType wouldn't correspond with folderId

Reply:
Oh good catch. This should have referred to `getMimeType` (and even that needs a follow up comment because as I've learned with my more recent commit, that is in conflict with another interface that provides `getEncoding` (which is the same thing)).

Will fix before merging.

Reply:
Okay so because PhotoModel.java doesn't use MediaObject (and thus doesn't have a `getEncodingFormat`) I'm going to punt this issue and leave this as one of our TODOs to rethink some of our models.

In the meantime I've clarified the relationship of the newer, smaller `Fileable` to the older `MediaObject`